health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/ReviewFragment.kt

```diff
@@ -214,7 +215,6 @@ class ReviewFragment private constructor(
                     indicator.isVisible = pages.size > 1
                     pager.isUserInputEnabled = pages.size > 1
                 })
-                removePagerConstraint()
```

Comment:
[Note] Removed this because it was already being called before in `onViewCreated`.